src/jvm.rs

```diff
@@ -116,15 +110,29 @@ pub struct Jvm<'jvm> {
 impl<'jvm> Jvm<'jvm> {
     fn get_or_default_init_jvm() -> crate::GlobalResult<&'static JavaVM> {
         GLOBAL_JVM.get_or_try_init(|| {
-            let jvm = JvmBuilder::new().build_jvm()?;
+            let jvm = JvmBuilder::default_options().build_jvm()?;
```

Comment:
This should be a be a bit more likely to stay in sync with the documentation of `default_builder`.
```suggestion
            let jvm = Jvm::default_builder().build_jvm()?;
```